RunningOrder/Story/StoryDetail.swift

```diff
@@ -9,19 +9,26 @@
 import SwiftUI
 
 struct StoryDetail: View {
-    let story: Story
+    @Binding var story: Story
 
     var body: some View {
         VStack {
             Text(story.ticketReference)
             Text(story.name)
+            Text(story.environment)
```

Comment:
Do we have to keep this lines ?

Reply:
Just for test purpose to see change applied